remix.init/index.js

```diff
@@ -2,26 +2,80 @@ const { execSync } = require("child_process");
 const crypto = require("crypto");
 const fs = require("fs/promises");
 const path = require("path");
+
+const { toLogicalID } = require("@architect/utils");
 const inquirer = require("inquirer");
+const PackageJson = require("@npmcli/package-json");
 const YAML = require("yaml");
 
-const sort = require("sort-package-json");
-const { toLogicalID } = require("@architect/utils");
+const cleanupDeployWorkflow = (deployWorkflow, deployWorkflowPath) => {
+  delete deployWorkflow.jobs.typecheck;
+  deployWorkflow.jobs.deploy.needs = deployWorkflow.jobs.deploy.needs.filter(
+    (need) => need !== "typecheck"
+  );
 
-function getRandomString(length) {
-  return crypto.randomBytes(length).toString("hex");
-}
+  return [fs.writeFile(deployWorkflowPath, YAML.stringify(deployWorkflow))];
+};
+
+const cleanupVitestConfig = (vitestConfig, vitestConfigPath) => {
```

Comment:
If `vite-tsconfig-paths` is not needed, it should be removed.

Reply:
It is needed